manifests/settings.pp

```diff
@@ -41,4 +42,10 @@
       value => $email_smtp_password,
     }
   }
+
+  if $keycloak {
+    foreman_config_entry { 'authorize_login_delegation_auth_source_user_autocreate':
+      value =>  'External',
+    }
+  }
```

Comment:
This will not be a part of the PR anymore, am I correct @ekohl ?

Reply:
Yes, it was discussed to drop it.